### PR TITLE
Documentation (Probably should be proof-read first) 

### PR DESCRIPTION
**Pull request by kevinkarnani**

### CODE REVIEW

**Comment by GalaArc** [comment]
Will proofread tomorrow.

File: src/org/usfirst/frc/team321/subsystems/Intake.java [hunk 1]
```diff
@@ -8,23 +8,25 @@
 import edu.wpi.first.wpilibj.command.Subsystem;
 
 /**
-*Intake class for the intake of the boulders. The intake uses enums as set values for the
-*intake of boulders to either intake at max power, outtake at max power or just stall. 
+ * Intake class for the intake of the boulders. The intake uses enums as set values for the
+ * intake of boulders to either intake at max power, outtake at max power or just stall.
```
Comment (GalaArc) [inline src/org/usfirst/frc/team321/subsystems/Intake.java]: Make intake less repetitive. Barely understand.

File: src/org/usfirst/frc/team321/subsystems/Intake.java [hunk 2]
```diff
@@ -8,23 +8,25 @@
...
-*Intake class for the intake of the boulders. The intake uses enums as set values for the
-*intake of boulders to either intake at max power, outtake at max power or just stall. 
+ * Intake class for the intake of the boulders. The intake uses enums as set values for the
+ * intake of boulders to either intake at max power, outtake at max power or just stall. 
  */
 public class Intake extends Subsystem {
 	private SpeedController intakeMotor;
 	/**
-	 * Initializes Intake and calls the motor used for it from the RobotMap Class
+	 * Initializes Intake and calls the motor used for it from the RobotMap Class.
```
Comment (GalaArc) [inline src/org/usfirst/frc/team321/subsystems/Intake.java]: Remake so it says "calls the motor from the RobotMap Class"

File: src/org/usfirst/frc/team321/subsystems/Intake.java [hunk 3]
```diff
@@ -41,7 +43,9 @@ public double getValue(){
 	} 
 
 	/**
-	 * @param power new value of the intake within the limit using MathUtil
+	 * @param power If new value for the intake is required, 
+	 * will be within the limit using MathUtil's clamp method,
+	 * which creates a set range. Refer to documentation for more.
```
Comment (GalaArc) [inline src/org/usfirst/frc/team321/subsystems/Intake.java]: I dont get what this is saying

File: src/org/usfirst/frc/team321/subsystems/DriveTrain.java [hunk 4]
```diff
@@ -72,12 +77,14 @@ public double getRightEncoderValue() {
...
 	/**
 	 * Returns a normalized value in between a range of numbers
-	 * @param power The power to set the motor to
-	 * @param minPower The minimum power
-	 * @param maxPower The maximum power
-	 * @return
+	 *
+	 * @param power The power the motor is set to.
+	 * @param minPower The minimum power, which is -1.
+	 * @param maxPower The maximum power, which is 1.
```
Comment (GalaArc) [inline src/org/usfirst/frc/team321/subsystems/DriveTrain.java]: Remove "which is" from both lines

File: src/org/usfirst/frc/team321/utilities/JoystickUtil.java [hunk 5]
```diff
@@ -96,101 +108,108 @@ public static double getManiYValue() {
...
+	 * @return The angle created by the right joystick.
+	 *
 	 */
 	public static double getRightDegrees() {
 		return 180 - Math.toDegrees(Math.atan2(-getRightYValue(), -getRightXValue()));
 	}
 	
 	/**
-	 * Gives back the degrees of the joystick on the manipulator controller
+	 * Gives back the angle of the joystick on the manipulator controller.
```
Comment (GalaArc) [inline src/org/usfirst/frc/team321/utilities/JoystickUtil.java]: angle in degrees

File: src/org/usfirst/frc/team321/robot/Robot.java [hunk 6]
```diff
@@ -86,23 +86,27 @@ public void autonomousPeriodic() {
...
-        // continue until interrupted by another command, remove
-        // this line or comment it out.
         if (autonomousCommand != null) autonomousCommand.cancel();
     }
 
     /**
-     * This function is called periodically during operator control
+     * This function is called periodically during operator control. 
+     * The code creates an instance of each command, then runs it, which 
+     * will the desired state of the robot.
```
Comment (GalaArc) [inline src/org/usfirst/frc/team321/robot/Robot.java]: Questionable english, in other words what is this saying

**Comment by kevinkarnani** [comment]
Henry, I believe this conflict is created since Brian's PR went through. Should I refork and copy & paste the documentation over again, or is this conflict something else?

**Comment by kevinkarnani** [comment]
Also Junhao, how do you feel about the documentation now?

**Pull request closed without merging**